### PR TITLE
Add line number in comments for some functions (to support Mariposa's Verus integration features)

### PR DESCRIPTION
**Pull request by jetline0**
This feature is intended to make it easier for [Mariposa](https://github.com/secure-foundations/mariposa) to determine the original location of unstable Verus queries from their corresponding SMT2 files.

Specifically, the additional comment is for queries labeled as Function-Defs, Function-Decl-Check-Recommends, and Function-Terminations.
Ex:
```
;; Function-Def main::impl_u::l1::Directory::lemma_resolve_structure_assertions
;; /home/gelatin/verified-nrkernel/page-table/impl_u/l1.rs:744:5: 744:80 (#0)
```

A similar comment is already present in some SMT2 files generated by Verus, but the comment only appears in queries that use the spinoff prover:
Ex:
```
;; MODULE 'impl_u::l2_impl'
;; /home/gelatin/verified-nrkernel/page-table/impl_u/l2_impl.rs:358:9: 358:15 (#0)
```

### CODE REVIEW

File: source/rust_verify/src/verifier.rs [hunk 1]
```diff
@@ -591,7 +591,7 @@ impl Verifier {
         qid_map: &HashMap<String, vir::sst::BndInfo>,
         module: &vir::ast::Path,
         function_name: Option<&Fun>,
-        comment: &str,
+        comment: (&str, &str),
```
Comment (utaal) [inline source/rust_verify/src/verifier.rs:594]: Can you take the span as a new parameter instead, and stringify it here instead? This way we cannot forget to add it if we add new calls to `run_commands_queries`.

**Review by utaal — CHANGES_REQUESTED** [review]
This looks fine to me, modulo one hopefully minor change. Thanks!

File: source/rust_verify/src/verifier.rs (hunk 1 above)
Comment (jetline0) [reply source/rust_verify/src/verifier.rs]: I think the commit below should fix this problem! Turns out `commands_with_context` contains the span information I needed, so no span information needs to be passed into `run_commands_queries.`

**Comment by utaal** [comment]
Can you please run `vargo fmt`?

**Comment by utaal** [comment]
Thank you!

**Pull request merged**